test/js/inline-style-property-map.js

```diff
@@ -0,0 +1,15 @@
+suite('Inline StylePropertyMap', function() {
+  setup(function() {
+    this.element = document.createElement('div');
+    document.body.appendChild(this.element);
+    this.element.style.opacity = '0.5';
+  });
+  teardown(function() {
+    document.body.removeChild(this.element);
+  });
+
+  test('Test that a new InlineStylePropertyMap is created when a HTML element referance calls the styleMap method', function() {
```

Comment:
s/InlineStylePropertyMap/StylePropertyMap

I would change this to "Element.styleMap returns a StylePropertyMap."